NuGet/Microsoft.ChakraCore.Debugger.nuspec

```diff
@@ -11,12 +14,18 @@
     <description>Debugging companion library for the ChakraCore JavaScript engine.</description>
     <authors>Microsoft</authors>
     <projectUrl>https://github.com/microsoft/ChakraCore-Debugger</projectUrl>
-    <licenseUrl>https://github.com/microsoft/ChakraCore-Debugger/blob/master/LICENSE</licenseUrl>
+    <repository type="git" url="$repoUrl$" branch="$repoBranch$" commit="$repoCommit$" />
+    <license type="file">LICENSE.txt</license>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <copyright>Copyright © 2019 Microsoft</copyright>
+    <copyright>Copyright &#169; 2019 Microsoft</copyright>
```

Comment:
Looking at the license change, should the year be updated to 2020?

Reply:
Not sure. If so, it can be done in a subsequent change.